From c62da06e99a2bd1b66961ff60fa43dd119af5a01 Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Tue, 17 Sep 2019 20:49:00 +0300
Subject: [PATCH 5/5] Add pru to compare-all-tests

contrib/ChangeLog:

2019-10-13  Dimitar Dimitrov  <dimitar@dinux.eu>

	* compare-all-tests (all_targets): Add pru target.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 contrib/compare-all-tests | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/compare-all-tests b/contrib/compare-all-tests
index 502cc64f522..6d0f29e052f 100644
--- a/contrib/compare-all-tests
+++ b/contrib/compare-all-tests
@@ -34,7 +34,7 @@ s390_opts='-m31 -m31/-mzarch -m64'
 sh_opts='-m3 -m3e -m4 -m4a -m4al -m4/-mieee -m1 -m1/-mno-cbranchdi -m2a -m2a/-mieee -m2e -m2e/-mieee'
 sparc_opts='-mcpu=v8/-m32 -mcpu=v9/-m32 -m64'
 
-all_targets='alpha arm avr bfin cris fr30 frv h8300 ia64 iq2000 m32c m32r m68k mcore mips mmix mn10300 pa pdp11 ppc sh sparc v850 vax xstormy16 xtensa' # e500 
+all_targets='alpha arm avr bfin cris fr30 frv h8300 ia64 iq2000 m32c m32r m68k mcore mips mmix mn10300 pa pdp11 ppc pru sh sparc v850 vax xstormy16 xtensa' # e500
 
 test_one_file ()
 {
-- 
2.20.1

